examples/functional_usage/train.py

```diff
@@ -0,0 +1,41 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+import torch
+import transformers
+
+from renate.benchmark.datasets.nlp_datasets import HuggingFaceTextDataModule
+from renate.models.renate_module import RenateWrapper
+from renate.updaters.experimental.fine_tuning import FineTuningModelUpdater
+
+
+### Create model.
+transformer_model = transformers.DistilBertForSequenceClassification.from_pretrained(
+    "distilbert-base-uncased", num_labels=2, return_dict=False
+)
+model = RenateWrapper(transformer_model)
```

Comment:
It's not strictly related to this PR but it would be nice to discuss the pros and cons of doing this operation directly in the model updater.